pkg/controller/servicebindingrequest/olm.go

```diff
@@ -250,15 +289,15 @@ func buildDescriptorsFromAnnotations(annotations map[string]string) (
 		sort.Strings(descriptors)
 		path := strings.Split(fieldPath, ".")
 		if path[0] == "status" {
-			currentStatusDescriptor = &olmv1alpha1.StatusDescriptor{
+			currentStatusDescriptor = append(currentStatusDescriptor, olmv1alpha1.StatusDescriptor{
```

Comment:
@baijum @isutton This fixes a bug. Without this, only 1 spec descriptor and 1 status descriptor was being generated from multiple annotations, causing us to lose information.

This was never caught because determinations from annotations are probably being over-witten by information in the metadata.

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/371